examples/pytorch/graphsaint/README.md

```diff
@@ -26,74 +26,120 @@ All datasets used are provided by Author's [code](https://github.com/GraphSAINT/
 
 Note that the PPI dataset here is different from DGL's built-in variant.
 
+## Config
+
+- The config file is `config.py`, which contains best configs for experiments below.
+- Please refer to `sampler.py` to see explanations of some key parameters.
```

Comment:
"Please refer to `sampler.py` to see explanations of some key parameters." This may not be sufficient. For example, 'aggr' is for model architecture and not in sampler.py.

Reply:
Should we need to put explanations of all parameters including that correlated to the model architecture in README.md? Actually the sampler.py only contains the parameters correlated to the sampler itself.

Reply:
Maybe we need to explain all parameters in `config.py` in README.md?

Reply:
I guess you can explain them either in README or config.py. For the latter one, just leave one comment per parameter.